src/test/resources/regressions/features/loops/range-fail2.gobra

```diff
@@ -0,0 +1,11 @@
+// Any copyright is dedicated to the Public Domain.
+// http://creativecommons.org/publicdomain/zero/1.0/
+
+package pkg
+
+func foo_arr() {
+	x := []uint{1, 2, 3, 4}
+	//:: ExpectedOutput(invariant_not_well_formed)
```

Comment:
Why is the invariant not well formed?

Reply:
One of the invariants that is added automatically is:
len(x) > 0 ==> 0 <= i && i < len(x) ==> j == x[i]
which fails because we don't have acc(x).
The user wouldn't be able anyway to do a range over this loop because inside the loop body, an assignment from x[i] would happen to j. The error message is not super clarifying though.

Reply:
Error message improved